shared/ff_util/include/ff_util/config_client.h

```diff
@@ -20,13 +20,18 @@
 #define FF_UTIL_CONFIG_CLIENT_H_
 
 #include <ff_common/ff_ros.h>
-
 #include <ff_common/ff_names.h>
 
 #include <string>
 #include <list>
 
 namespace ff_util {
+
+#ifndef FF_LOGGER
```

Comment:
I'd make this specific to the header in case you try to do it again somewhere else we will get very confused, like FF_LOGGER_CONFIG_CLIENT

Reply:
Thinking about this more, it  might still cause problems if we define another logger in a ff_util c++ file that includes this one. Probably we don't, but we could in the future. It might be better to make a cpp file error function like I did for config_server.

Reply:
Yeah the goal here is to import it only once across headers, because it was complaining about redefinition of _FF_LOGGER. 

Uhh I didn't see your Error function, I'll do that.